File: 1297-maximum-number-of-balloons/maximum-number-of-balloons.py
class Solution:
    def maxNumberOfBalloons(self, text: str) -> int:
        MAP = Counter(text)
        bC = MAP.get('b',0)
        aC = MAP.get('a',0)
        nC = MAP.get('n',0)
        lC = MAP.get('l',0)//2
        oC = MAP.get('o',0)//2
        return min(bC,aC,nC,lC,oC)

File: 1297-maximum-number-of-balloons/maximum-number-of-balloons.cpp
class Solution {
public:
    int maxNumberOfBalloons(string text) {
        unordered_map<char,int> MAP;
        for (char &ch: text) MAP[ch]++;
        int bC = MAP['b'];
        int aC = MAP['a'];
        int nC = MAP['n'];
        int lC = MAP['l']/2;
        int oC = MAP['o']/2;
        return min(min(min(bC,aC),nC),min(lC,oC));
    }
};